invenio_communities/models.py

```diff
@@ -240,6 +256,20 @@ def get(cls, community_id, with_deleted=False):
         return q.one_or_none()
 
     @classmethod
+    def get(cls, community_id=None, community_name=None, with_deleted=False):
+        """Get a community."""
+        assert((community_id is not None) ^ (community_name is not None),
```

Comment:
Sorry, I thought I had removed it, this one is just extending the previous implementation.